### ISSUE TITLE
f.cam6_3_132.FMTHIST_ne30.sci_opt_roughTopo.001

### ISSUE THREAD

**Issue by PeterHjortLauritzen** [NCAR/amwg_dev#473]
**Description:**

Investigate if rougher topography has an impact on stratospheric wind biases in FMT.

Same setup as "science optimized" code (https://github.com/NCAR/amwg_dev/issues/470) but using a smoothing radius of 50km instead of 100km when smoothing PHIS.

Command used to generate PHIS:

qcmd -l walltime=4:00:00 -l select=1:ncpus=1 -- ./cube_to_target --grid_descriptor_file ../regression-test-data/ne30pg3.nc --intermediate_cs_name /glade/campaign/cgd/amp/pel/topo/cubedata/gmted2010_modis_bedmachine-ncube3000-220518.nc --grid_descriptor_file_gll ../regression-test-data/ne30np4.nc --output_grid ne30pg3 --smoothing_scale 50.0 --name_email_of_creator 'Peter Hjort Lauritzen, pel@ucar.edu'

___
**Case directory:**
/glade/campaign/cesm/cesmdata/cseg/runs/cesm2_0/f.cam6_3_132.FMTHIST_ne30.sci_opt_roughTopo.001

- On github:
https://github.com/NCAR/amwg_dev/tree/$CASE
___
**Sandbox:**
- Locally (if still available):
/glade/work/hannay/cesm_tags/cam6_3_132 with code mods from https://github.com/NCAR/amwg_dev/issues/470

- On github:
https://github.com/ESCOMP/CAM/tree/cam6_3_132
___
**Diagnostics:**
- ADF diags (if available)
[https://webext.cgd.ucar.edu/FLTHIST/$CASE/atm/](https://webext.cgd.ucar.edu/FLTHIST/$CASE/atm/)
___
**Output:**
/glade/derecho/scratch/pel/archive/f.cam6_3_132.FMTHIST_ne30.sci_opt_roughTopo.001/
___
**Contacts:**
@cecilehannay @PeterHjortLauritzen @JulioTBacmeister @tilmes 
___
**Extra details:**

**Comment by PeterHjortLauritzen** [comment]
PHIS difference between rougher topography and default:

<img width="575" alt="Screenshot 2024-01-18 at 11 18 16 AM" src="https://github.com/NCAR/amwg_dev/assets/8782989/af8dcb93-86f4-42f4-8d00-1f443a248ce7">

(Colorbar is from -4000 to 6000)

Note ~600m difference in elevation near Antarctic peninsular, Southern Greenland, ...

In terms of numerical noise the rougher topography does not seem be an issue in terms of common fields I look at:

10 year average of OMEGA500 (rough topo left and default right):

<img width="1137" alt="Screenshot 2024-01-18 at 11 23 55 AM" src="https://github.com/NCAR/amwg_dev/assets/8782989/f490427d-6daf-4f7d-bfc0-5d339743a68c">

10 year average of PSL:

<img width="1133" alt="Screenshot 2024-01-18 at 11 25 56 AM" src="https://github.com/NCAR/amwg_dev/assets/8782989/7dec297e-b2a2-4cc9-bf80-bba3d8dd9385">

10 year average of absolute value of surface pressure tendency:

<img width="1142" alt="Screenshot 2024-01-18 at 11 27 18 AM" src="https://github.com/NCAR/amwg_dev/assets/8782989/8ba2483e-f292-45c2-8bd2-4516e9841724">

**Comment by JulioTBacmeister** [comment]
The rougher topo looks like it's driving things in the right direction in the southern hemisphere. It appears more drag is still needed though.

![image](https://github.com/NCAR/amwg_dev/assets/9221710/d71c7754-9cea-4d6a-9b88-1916d1dc3c4e)

**Comment by islasimpson** [comment]
Here's some aspects of the zonal mean zonal wind.  I think the effect on the SH is pretty minimal.


![uzm](https://github.com/NCAR/amwg_dev/assets/41842751/f5a0a670-664d-4f97-8a45-90419cd99be3)
![ush](https://github.com/NCAR/amwg_dev/assets/41842751/e4880ba9-9c86-4567-8bae-bce232a33d62)

**Comment by islasimpson** [comment]
Sorry, forgot the legend in the above plot. blue is ctrl and red is rough in the above plot.  ERA5 is black.

**Comment by JulioTBacmeister** [comment]
It seems the rougher topography is a small step in the right direction in the SH and doesn't do noticeable harm in the NH. @adamrher is the dual polar grid easy to use? I think it might be interesting to see what the SH looks like in that configuration.

**Comment by adamrher** [comment]
Yes. I can setup a sandbox for you with it installed. What cam tag do u want me to use?

**Comment by PeterHjortLauritzen** [comment]
We are using cam6_3_132 for this experiment

**Comment by JulioTBacmeister** [comment]
only question is do we redo the topo for dual polar to be rougher?

**Comment by PeterHjortLauritzen** [comment]
Since it did not help much I think we should just use default ... I suppose the hypothesis is whether resolution is the issue!

**Comment by adamrher** [comment]
Interesting improvements in Greenland precip in the rough topo experiment, mainly the reduction in the North.

![temp_contours precip](https://github.com/NCAR/amwg_dev/assets/29961476/a1ceabeb-8711-4871-958c-191ed79ca251)

**Comment by JulioTBacmeister** [comment]
Promising. Any precip change around Antarctica?

**Comment by adamrher** [comment]
> Promising. Any precip change around Antarctica?

It seems a bit improved over Antarctica as well, compared to RACMO.

![temp_contours](https://github.com/NCAR/amwg_dev/assets/29961476/34d2a741-27a5-4431-a91b-00f06a8ab2f8)
